specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/virtualMachineScaleSet.json

```diff
@@ -103,6 +103,22 @@
             "type": "string",
             "description": "The name of the VM scale set to create or update."
           },
+          {
+            "name": "If-Match",
```

Comment:
please add these at the end of the "parameters" to resolve the breaking change flag